app/screens/Observations/map.js

```diff
@@ -100,8 +103,38 @@ class ObservationMapScreen extends Component {
     this.refs["menuInstance"].setVelocity({ x: 3000 });
   }
 
-  componentDidMount() {
-    console.log("map", this._map);
+  getFeatures() {
+    this._map.getBounds(data => {
+      var q = [[data[0], data[2]], [data[1], data[3]]];
+      var annotations = [];
+      var stream = this.osm.queryGeoJSONStream(q);
```

Comment:
yeah, i'd like to abstract this out more to make it reusable in other views. figured that could happen in upcoming prs.